constants.yml

```diff
@@ -61,8 +61,8 @@ company_type:
     'icvc-securities' : "Investment company with variable capital"
     'icvc-warrant' : "Investment company with variable capital"
     'icvc-umbrella' : "Investment company with variable capital"
-    'registered-society-non-jurisdictional': "Registered society"
-    'industrial-and-provident-society' : "Industrial and provident society"
+    'registered-society-non-jurisdictional': "Registered Society"
+    'industrial-and-provident-society' : "Industrial and Provident society"
```

Comment:
Capital `S` in `society`?

Reply:
Yes - this is the requirement - capitalisation of those characters